iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/functionclass/EventDataListener.java

```diff
@@ -48,6 +52,7 @@ public EventDataListener(final Device device, final Scheduler scheduler, final D
     public List<ServiceError> write(final List<BasicDataAttribute> bdas) {
         logger.info("BDA write request scanning...");
         boolean modified = false;
+        HashMap<RelaySchedulePair, Schedule> toUpdateSchedules = new HashMap<RelaySchedulePair, Schedule>();
```

Comment:
Change this to:
Map<RelaySchedulePair, Schedule> toUpdateSchedules = new HashMap<>();

1) Declaring a variable as an interface (like Map) makes it slightly easier to maintain.
2) Just using "<>" when constructing the HashMap saves some typing.

Reply:
Done